package.json

```diff
@@ -52,7 +55,8 @@
     "eslint-plugin-ember": "^5.2.0",
     "eslint-plugin-node": "^7.0.1",
     "loader.js": "^4.7.0",
-    "qunit-dom": "^0.8.0"
+    "qunit-dom": "^0.8.0",
```

Comment:
Do we use `qunit-dom` actually?

Reply:
Yep, for asserting attributes and classes, maybe something else.